src/meta/src/barrier/recovery.rs

```diff
@@ -80,7 +80,7 @@ where
             let mut info = self.resolve_actor_info_for_recovery().await;
             let mut new_epoch = prev_epoch.next();
 
-            if self.enable_migrate {
+            {
                 // Migrate expired actors to newly joined node by changing actor_map
                 self.migrate_actors(&info).await?;
```

Comment:
Add a bool flag return param for `migrate_actors` to decide whether we need to resolve actor info or not.